broker/src/tunneldigger_broker/main.py

```diff
@@ -91,11 +91,18 @@
 broker_host = config.get('broker', 'address')
 for port in config.get('broker', 'port').split(','):
     try:
-        broker_instance = broker.Broker(
-            (broker_host, int(port)),
-            config.get('broker', 'interface'),
-            tunnel_manager,
-        )
+        if config.has_option('broker', 'interface'):
```

Comment:
Changing whether an option is optional should be reflected in the docs and/or example config file as well.